_policies/triage-bugzilla.md

```diff
@@ -28,20 +28,19 @@ If you need to change who is responsible for triaging a bug in a component, plea
 
 As a triage owner the queries you should be following for your component are:
 
-* All open bugs in your component without a pending needinfo flag since start of current release cycle
-* All open bugs with pending needinfo flags in your component which have not been modified in two weeks
+* All open bugs of type `defect` in your component without a pending `needinfo` flag or a `stalled` keyword since start of current release cycle which do not have a priorty set
+* All open bugs of type `defect` with pending needinfo flags in your component which have not been modified in two weeks
+* All bugs with active review requests in your component which have not been modified in five days
 
```

Comment:
Also, I would like triage owner to act when a needinfo request in a bug has been unanswered for more than 10 days. Many bugs get stalled with a last comment ending on a NeedInfo that doesn't get answered for weeks or months.